### PR TITLE
Update README to be up-to-date

### PR DESCRIPTION
**Pull request by sanposhiho**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Update README to be up-to-date

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
+- configure with [KubeSchedulerConfiguration](https://kubernetes.io/docs/reference/scheduling/config/)
+- add Plugins of [Scheduling Framework](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/)
+- add [Extenders](https://github.com/kubernetes/enhancements/tree/5320deb4834c05ad9fb491dcd361f952727ece3e/keps/sig-scheduling/1819-scheduler-extender)
+- etc...
+
+But, unfortunately, not all configuration/expansions yield good results.
+Those who customize the scheduler need to make sure their scheduler is working as expected, and doesn't have an unacceptably negative impact on the scheduling. 
+
+In real Kubernetes, we cannot know the results of scheduling in detail without reading the logs, which usually requires privileged access to the control plane.
+That's way we are developing a simulator for kube-scheduler -- you can try out the behavior of the scheduler with web UI while checking which plugin made what decision for which Node.
```
Comment (196Ikuchil) [inline README.md:15]:
```suggestion
But, unfortunately, not all configurations/expansions yield good results.
Those who customize the scheduler need to make sure their scheduler is working as expected, and doesn't have an unacceptably negative impact on the scheduling. 

In real Kubernetes, we cannot know the results of scheduling in detail without reading the logs, which usually require privileged access to the control plane.
That's way we are developing a simulator for kube-scheduler -- you can try out the behavior of the scheduler with web UI while checking which plugin made what decision for which Node.
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,42 @@
...
+
+## Simulator's architecture
+
+We have several components:
+- simulator (in `/simulator`)
+- Web UI (in `/web`)
+- coming soon... :)  (see [./keps](./keps) to see some nice ideas we're working on)
```
Comment (196Ikuchil) [inline README.md:22]:
```suggestion
- Simulator (in `/simulator`)
- Web UI (in `/web`)
- Coming soon... :)  (see [./keps](./keps) to see some nice ideas we're working on)
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,42 @@
...
+Simulator internally has kube-apiserver, scheduler, and HTTP server for web UI.
+
+You can create any resources by communicating with kube-apiserver via kubectl, k8s client library, or web UI.
+
+See the following docs to know more about simulator:
+- [how-it-works.md](simulator/docs/how-it-works.md): describes about how the simulator works.
+- [kube-apiserver.md](simulator/docs/kube-apiserver.md): describe about kube-apiserver in simulator. (how you can configure and access) 
+- [api.md](simulator/docs/api.md): describes about HTTP server the simulator has.
+
+### web
```
Comment (196Ikuchil) [inline README.md:35]:
```suggestion
### Web UI
```
We should unify the style?

**Review by 196Ikuchil — COMMENTED** [review]
overall looks good~
(small nits)

**Comment by sanposhiho** [comment]
@196Ikuchil Fixed :)

**Comment by 196Ikuchil** [comment]
👌
/lgtm

**Pull request merged**